Packs/Clarizen/Integrations/ClarizenIAM/ClarizenIAM.py

```diff
@@ -0,0 +1,414 @@
+import demistomock as demisto
+from CommonServerPython import *
+from IAMApiModule import *
+import traceback
+import urllib3
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+USER_FIELDS = 'Name,Email,Region,Location,JobTitle,DirectManager,MobilePhone,TimeZone,username,profile,firstname,' \
+              'lastname,state'
+ERROR_CODES_TO_SKIP = [
+    404
+]
+
+
+'''CLIENT CLASS'''
+
+
+class Client(BaseClient):
+    """ A client class that implements logic to authenticate with the application. """
+
+    def __init__(self, base_url, verify=True, proxy=False, ok_codes=tuple(), headers=None, auth=None):
+        super().__init__(base_url, verify, proxy, ok_codes, headers, auth)
+        self.username = auth[0]
+        self.password = auth[1]
+
+    def test(self):
+        """ Tests connectivity with the application. """
+
+        uri = '/authentication/login'
+        params = {
+            "userName": self.username,
+            "Password": self.password
+        }
+        
+        self._http_request(method='GET', url_suffix=uri, params=params)
+
+    def get_user_by_id(self, user_id):
+        uri = f'/data/objects/{user_id}'
+        params = {
+            'fields': USER_FIELDS
```

Comment:
@anara123 should we keep this hard-coded, or should we get the full list from the get_app_fields() response?